frontend/src/game/Cols.scss

```diff
@@ -20,4 +20,27 @@
       }
     }
   }
+
+  .helper {
```

Comment:
I'd like this to be called something more specific so the styles never collide.
Let's rename the component something like `CornerCardPreview` or something, AND call the class the same thing (caps and all) . Makes finding it in the code base a breeze